src/gt4py/cartesian/backend/dace_backend.py

```diff
@@ -56,17 +56,17 @@
 
 
 def _specialize_transient_strides(sdfg: dace.SDFG, layout_map):
-    repldict = replace_strides(
+    replacement_dictionary = replace_strides(
```

Comment:
Just an optional suggestion: since the method where this is used is called `replace_dict` maybe it'd be better to call the variable `replacement_dict` so they both method and variable use the same suffix.